<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Таблица коэффициентов</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите точки:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Построить</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>171</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLineEdit" name="lineEditX7">
        <property name="text">
         <string>12.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX8">
        <property name="text">
         <string>18.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX9">
        <property name="text">
         <string>24.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX10">
        <property name="text">
         <string>36.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX11">
        <property name="text">
         <string>48.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX12">
        <property name="text">
         <string>72.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLineEdit" name="lineEditY7">
        <property name="text">
         <string>26.63</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY8">
        <property name="text">
         <string>19.40</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY9">
        <property name="text">
         <string>13.26</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY10">
        <property name="text">
         <string>5.88</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY11">
        <property name="text">
         <string>2.56</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY12">
        <property name="text">
         <string>0.49</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>171</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLineEdit" name="lineEditX1">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX2">
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX3">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX4">
        <property name="text">
         <string>2.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX5">
        <property name="text">
         <string>4.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditX6">
        <property name="text">
         <string>8.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLineEdit" name="lineEditY1">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY2">
        <property name="text">
         <string>5.36</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY3">
        <property name="text">
         <string>9.35</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY4">
        <property name="text">
         <string>17.18</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY5">
        <property name="text">
         <string>25.78</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditY6">
        <property name="text">
         <string>29.78</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>21</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string> 7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string> 8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string> 9</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>11</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>12</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Настройка осей координат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>310</y>
     <width>26</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ox:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOx1">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>310</y>
     <width>52</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-10.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>16</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>до</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOx2">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>310</y>
     <width>53</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>80.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>16</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>от</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>16</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>от</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOy2">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>310</y>
     <width>52</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>35.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>310</y>
     <width>26</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Oy:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOy1">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>310</y>
     <width>52</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-5.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>16</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>до</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>16</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
